### ISSUE TITLE
Open sourced RT-DETRv4 Finetuned on VisDrone Aerial Imagery Dataset

### ISSUE THREAD

**Issue by dronefreak** [RT-DETRs/RT-DETRv4#42]
Hello @Zhao-Yian @zjliao25 @JTroyerOvermatch 

First of all, congratulations and thank you for open sourcing this great work! 

I have been playing around with the repo a little bit and I have been able to finetune the s/m/l variants on the VisDrone Aerial Imagery Dataset for aerial object detection, crowd tracking etc.

Here is a small [demo](https://huggingface.co/dronefreak/visdrone-rtdetrv4-l/blob/main/assets/demo_banner.mp4) using the RT-DETRv4-L variant.

## VisDrone RT-DETRv4 Model Zoo

RT-DETRv4 model fine-tuned and evaluated on VisDrone test set

| Model                | mAP@50        | mAP@50-95       | Precision         | Recall         |
| --------------------- | ------------- | --------------- | ----------------- | -------------- |
| [RT-DETRv4-L](https://huggingface.co/dronefreak/visdrone-rtdetrv4-l) | 48.32 | 28.89 | 56.37 | 50.71 |
| [RT-DETRv4-M](https://huggingface.co/dronefreak/visdrone-rtdetrv4-m) | 46.95 | 27.81 | 55.62 | 48.95 |
| [RT-DETRv4-S](https://huggingface.co/dronefreak/visdrone-rtdetrv4-s) | 46.28 | 27.36 | 55.42 | 49.17 |

Thank you again for open sourcing this work!
